packages/tokens/src/tokens.scss

```diff
@@ -152,6 +152,8 @@
   --text-overline-letter: 0.014em;
   --text-badge: var(--font-weight-bold) 10px/16px var(--font-family);
   --text-badge-letter: 0.0004em;
+  --text-tag: var(--font-weight-bold) 12px/20px var(--font-family);
```

Comment:
(question) does this exist in the figma token specs? If you don't need to decide with the ux team, open a thread on `#chapter-ux`

Reply:
@felipefialho [i created this thread](https://juntossomosmais.slack.com/archives/C0117K4MYSW/p1694631760012019) so we can decide with the design team

Reply:
thanks! let's wait

Reply:
@felipefialho we decided to fix the weight for the `text-caption-focused` tokens so I could remove the previous tokens that I had created https://github.com/juntossomosmais/atomium/pull/261/commits/8b06064ad92f8216cd0a487c1e74461528b68c6d